src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/RelevantEndOfSentenceDataload.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDate
+
+@Entity
+@Table(name = "relevant_sentence_end_dataload")
+class RelevantEndOfSentenceDataload(
```

Comment:
Hi @awhodgson256 , did you contemplated on creating a csv for this rather than a table. The reason why I am thinking this is a one off job and reading from csv would have been easier

Reply:
Ok @awhodgson256 , no worries.